v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/ExtractionPipeline.scala

```diff
@@ -36,12 +36,31 @@ object ExtractionPipeline {
   def main(rawArgs: Array[String]): Unit = {
     val (pipelineContext, parsedArgs) = ContextAndArgs.typed[Args](rawArgs)
 
+    val inputDir = parsedArgs.inputDir
+    val outputDir = parsedArgs.outputDir
```

Comment:
Do we need these locals, since they're only used once?

Reply:
Nope